nsxt/resource_nsxt_compute_manager.go

```diff
@@ -0,0 +1,533 @@
+			"tag":          getTagsSchema(),
+			"access_level_for_oidc": {
+				Type:         schema.TypeString,
+				Description:  "Specifies access level to NSX from the compute manager",
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice(accessLevelForOidcValues, false),
+				Default:      model.ComputeManager_ACCESS_LEVEL_FOR_OIDC_FULL,
+			},
+			"create_service_account": {
+				Type:        schema.TypeBool,
+				Description: "Specifies whether service account is created or not on compute manager",
+				Optional:    true,
+				Default:     false,
+			},
+			"credential": {
+				Type:        schema.TypeList,
+				Description: "Login credentials for the compute manager",
+				Required:    true,
+				MaxItems:    1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"saml_login": {
+							Type:        schema.TypeList,
+							Description: "A login credential specifying saml token",
+							Optional:    true,
+							MaxItems:    1,
+							ExactlyOneOf: []string{
+								"credential.0.saml_login",
+								"credential.0.session_login",
+								"credential.0.username_password_login",
+								"credential.0.verifiable_asymmetric_login",
+							},
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"thumbprint": {
+										Type:        schema.TypeString,
+										Description: "Thumbprint of the server",
+										Required:    true,
+									},
+									"token": {
+										Type:        schema.TypeString,
+										Description: "The saml token to login to server",
+										Required:    true,
+										Sensitive:   true,
+									},
+								},
+							},
+						},
+						"session_login": {
+							Type:        schema.TypeList,
+							Description: "A login credential specifying session_id",
+							Optional:    true,
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"session_id": {
+										Type:        schema.TypeString,
+										Description: "The session_id to login to server",
+										Required:    true,
+										Sensitive:   true,
+									},
+									"thumbprint": {
+										Type:        schema.TypeString,
+										Description: "Thumbprint of the login server",
+										Required:    true,
+									},
+								},
+							},
+						},
+						"username_password_login": {
+							Type:        schema.TypeList,
+							Description: "A login credential specifying a username and password",
+							Optional:    true,
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"password": {
+										Type:        schema.TypeString,
+										Description: "The authentication password for login",
+										Required:    true,
+										Sensitive:   true,
+									},
+									"thumbprint": {
+										Type:        schema.TypeString,
+										Description: "Thumbprint of the login server",
+										Required:    true,
```

Comment:
`thumbprint` is not required in UI

Reply:
Indeed - instead the UI fetches the `thumbprint` and asks for verification.
with `curl` the transaction fails when the `thumbprint` is missing with the following error:

`"Compute manager server x.x.x.x thumbprint input empty is not valid. Current retrieved compute manager server thumbprint is xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx . If correct, please re-submit with this thumbprint"`